integration_tests/projects/013_structured_hooks/scripts/types.py

```diff
@@ -0,0 +1,8 @@
+from fal.typing import *
+from _fal_testing import create_dynamic_artifact
+
+create_dynamic_artifact(
+    context,
+    "Types: "
+    + ", ".join(f"{key}={type(value).__name__}" for key, value in context.arguments.items()),
```

Comment:
can we also print `value`?

```suggestion
    + ", ".join(f"{key}={value} ({type(value).__name__})" for key, value in context.arguments.items()),
```

Reply:
sure!